src/sentry/runner/commands/backup.py

```diff
@@ -14,7 +14,13 @@
 @click.argument("src", type=click.File("rb"))
 @configuration
 def import_(src):
-    "Imports data from a Sentry export."
+    """CLI command wrapping the `importer` functionality."""
+
+    importer(src)
```

Comment:
this shouldn't be needed -- you can call click functions directly (it's ~sorta the point of why it's a decorator-based framework)

Reply:
We're planning to allow importing from an API endpoint in the near future, which means "import this data" will be called from multiple paths. It seems unlikely those paths will share all of their arguments - there are probably only some that we want to expose via the CLI, so it makes sense to have a wrapper function to handle that particular use case.

Reply:
do that _when you need it_ rather than _in case you need it_ -- it increases the scope of this PR and may never be needed

Reply:
Done.